src/component/PrintPanel/PrintPanelV3.tsx

```diff
@@ -225,9 +226,8 @@ export class PrintPanelV3 extends React.Component<PrintPanelV3Props, PrintPanelV
           scales: printManager.getScales(),
           dpis: printManager.getDpis(),
           outputFormats: printFormats,
-          // TODO Don't access private property ol_uid
-          // @ts-ignore
-          legendIds: this.getFilteredLegendLayers().map((layer) => layer.ol_uid)
+          // TODO double check whether casting to Number is better than changing the type
+          legendIds: this.getFilteredLegendLayers().map((layer) => Number(getUid(layer)))
```

Comment:
Technically uid will always be a number, but this is an implementation detail of openlayers so it is better in my opinion to either change the type `legendIds` to `string` or `number|string`.